Dockerfile.no-systemd

```diff
@@ -10,6 +10,7 @@ RUN install_packages \
 		netcat \
 		net-tools \
 		ifupdown \
+    postgresql-client-11 \
```

Comment:
@jviotti indentation is wrong here

Reply:
@roman-mazur Fixed! (we should add an `.editorconfig` btw)